### PR TITLE
[Integration] - omevv_baseline_profile_info

### PR DESCRIPTION
**Pull request by ShivamSh3**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: tests/integration/targets/omevv_baseline_profile_info/tests/all_omevv_baseline_profile_info.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  ansible.builtin.fail:
+    msg: "Please define the following variables: hostname, vcenter_username
+      and vcenter_password."
+  when:
+    hostname is not defined or ome_username is not defined or ome_password
```
Comment (Kritika-Bhateja-03) [inline tests/integration/targets/omevv_baseline_profile_info/tests/all_omevv_baseline_profile_info.yaml]: It should be vcenter_username is undefined or vcenter_password is undefined

Comment (Kritika-Bhateja-03) [reply tests/integration/targets/omevv_baseline_profile_info/tests/all_omevv_baseline_profile_info.yaml]: Check in other files as well.

**Review by Kritika-Bhateja-03 — CHANGES_REQUESTED** [review]

File: tests/integration/targets/omevv_baseline_profile_info/tests/all_omevv_baseline_profile_info.yaml (hunk 1 above)
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/all_omevv_baseline_profile_info.yaml]: done

File: tests/integration/targets/omevv_baseline_profile_info/tests/__helper.yml
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/__helper.yml:1]: Change the file nae to _helper.yaml

File: tests/integration/targets/omevv_baseline_profile_info/tests/__pre_req_create.yml
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/__pre_req_create.yml:1]: change the file name to _pre_req_create.yaml

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        hostname: "invalid"
+      ignore_unreachable: true
+      register: invalid_hostname
+
+    - name: Verify task status- Negative - Invalid hostname
+      ansible.builtin.assert:
+        that:
+          - invalid_hostname.unreachable
+          - not invalid_hostname.changed
+          - invalid_hostname.msg == "<urlopen error [Errno -2]
```
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:27]: Assertion is not correct. kindly check based on the latest changes

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            Name or service not known>"
+
+    - name: Negative - Invalid vcenter_username
+      dellemc.openmanage.omevv_baseline_profile_info:
+        hostname: "{{ hostname }}"
+        vcenter_username: invalid
```
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:33]: Add assertion

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        vcenter_password: "{{ vcenter_password }}"
+      ignore_errors: true
+      register: invalid_user
+
+    - name: Verify task status - Negative - Invalid user
+      ansible.builtin.assert:
+        that:
+          - invalid_user.failed
+          - not invalid_user.changed
+          - 'invalid_user.msg == "HTTP Error 401: 401" or
```
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:43]: Assertion is not correct. kindly check based on the latest changes

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+        vcenter_password: invalid
+      ignore_errors: true
+      register: invalid_vcenter_password
+
+    - name: Verify task status - Negative - Invalid vcenter_password
+      ansible.builtin.assert:
+        that:
+          - invalid_vcenter_password.failed
+          - not invalid_vcenter_password.changed
+          - 'invalid_vcenter_password.msg == "HTTP Error 401: 401" or
```
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:59]: Assertion is not correct. kindly check based on the latest changes

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        invalid_validate_certs_expected_msg: "argument 'validate_certs' is of
+          type <class 'str'> and we were unable to convert to bool: The value
+          'invalid' is not a valid boolean.  Valid booleans include: "
+
+    - name: Verify task status - Negative - Invalid validate_certs
+      ansible.builtin.assert:
+        that:
+          - invalid_validate_certs.failed
+          - not invalid_validate_certs.changed
+          - invalid_validate_certs_expected_msg in invalid_validate_certs.msg
```
Comment (rajshekarp87) [inline tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:98]: Assert exact message

File: tests/integration/targets/omevv_baseline_profile_info/tests/__helper.yml
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/__helper.yml:1]: done

File: tests/integration/targets/omevv_baseline_profile_info/tests/__pre_req_create.yml
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/__pre_req_create.yml:1]: done

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml (hunk 2 above)
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:27]: checked, it is proper only

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml (hunk 3 above)
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:33]: checked, it is proper only

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml (hunk 4 above)
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:43]: checked, it is proper only

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml (hunk 6 above)
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:98]: checked, it is proper only

File: tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml (hunk 5 above)
Comment (ShivamSh3) [reply tests/integration/targets/omevv_baseline_profile_info/tests/invalid.yaml:59]: checked, it is proper only

**Comment by ABHISHEK-SINHA10** [comment]
Re-run the negative test once after merging collections in your branch, invalid test cases value might differ after Saksham's change in rest_api.py

**Review by Saksham-Nautiyal — APPROVED** [review]

**Review by Kritika-Bhateja-03 — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Pull request merged**